torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -19,6 +29,25 @@ def get_native_backend_config_dict():
         # optional, output activation dtype
         "output_dtype": torch.quint8
     }
+
+    # input_activation: quint8, output_activation: quint8
+    default_op_quint8_dtype_config = {
+        # optional, input activation dtype
+        "input_dtype": torch.quint8,
+        # optional, output activation dtype
+        "output_dtype": torch.quint8,
+    }
+
+    default_op_fp16_dtype_config = {
+        # optional, input activation dtype
+        "input_dtype": torch.float16,
+        # optional, weight dtype
+        "weight_dtype": torch.float16,
+        # optional, output activation dtype
+        "output_dtype": torch.float16,
+    }
+    # END dtype configs
```

Comment:
nit: I wonder if we should define these configs outside of `get_native_backend_config_dict` to make it easier to read, maybe in a future PR, otherwise this function will grow really large

Reply:
Such that `get_native_backend_config_dict` will only look like this:
```
def get_native_backend_config_dict():
    """
    ...
    """
    return {
        # optional
        "name": "native",
        "configs": [
            linear_module_config,
            *default_op_configs,
            *default_op_fp16_configs,
             ...
        ],
    }
```
What do you think?

Reply:
yeah mean define everything outside? I think we may want to hide some of the implementation details, don't really want to expose everything in native.py, but we can expose the dtype configs, it might be useful for user as well

Reply:
I guess if we prefix these things with `_` it could work